doc/governance/adding_member.rst

```diff
@@ -55,7 +55,7 @@ First, the new member should update and retrieve the latest state digest via the
 
 .. code-block:: bash
 
-    $ curl https://<ccf-node-address>/gov/ack/update_state_digest  --cacert service_cert.pem --key new_member_privk.pem --cert new_member_cert.pem
+    $ curl -X POST https://<ccf-node-address>/gov/ack/update_state_digest --cacert service_cert.pem --key new_member_privk.pem --cert new_member_cert.pem
```

Comment:
Good spot! While here, we should also update the endpoint names above and below this to link to the OpenAPI docs (ie `:http:GET:`/gov/ack/update_state_digest` rather than ``/gov/ack/update_state_digest``).

And as a very minor thing, I'd prefer the url to be the first argument, because our `scurl` arg-parsing isn't quite as permissive as `curl`, so if this changed from `curl` to `scurl` for some reason it'd break.

```suggestion
    $ curl https://<ccf-node-address>/gov/ack/update_state_digest -X POST --cacert service_cert.pem --key new_member_privk.pem --cert new_member_cert.pem
```